examples/image-component/pages/background.tsx

```diff
@@ -9,14 +9,12 @@ const BackgroundPage = () => (
     <div className={styles.bgWrap}>
       <Image
         alt="Mountains"
-        src={mountains}
+        src={bgImage}
         placeholder="blur"
         quality={100}
-        fill
+        layout="fill"
         sizes="100vw"
-        style={{
-          objectFit: 'cover',
-        }}
```

Comment:
`bgImage` isn't defined. Also, this example uses the new `next/image` and the changes u made are for `next/legacy/image`

Reply:
Thanks for explaning, but I'm not sure why I'm using next/legacy/image since I just created my project recently but the example doesn't work for me, could you guide me a little more on this?

Reply:
you are using the latest version of `next` where `next/image` is `next/future/image` and `next/legacy/image` is `next/image`. So your code is correct but this example uses `canary` version of nextjs

Reply:
Take a look at `package.json` in this example. Its using `canary` which has upcoming changes that are not released to stable yet.

Reply:
Thank you for the explanation, really appreciate it :)